### DISCUSSION TITLE
VPN being flagged as bot.

### DISCUSSION THREAD

**Discussion by shopliftersoftheworldunite** [apernet/hysteria#1214] [General]
Hey, while using hysteria some websites flag it as a bot and do not allow the user to use their services unless they make an account,
is there any way to deal with this problem?

**Comment by meower1** [comment]
hi. thats not a hysteria/protocol based problem. many sites use certain parameters to rate the safety of a client which since ur using a proxy its most likely that the provider you're using has a dirty ip (meaning ip's of that provider has been used for bots/spam/etc) the solution to this is often to set your outbound to cloudflare warp which often provides cleaner ips

this script provides an easy setup of what i've mentioned
https://github.com/ReturnFI/Hysteria2

for manual setup you'd have to run cloudflare warp cli as a socks proxy and set the hysteria's outbound to the local proxy provided by warp

**Comment by ParsaJR** [comment]
![image](https://github.com/user-attachments/assets/06da1795-a115-4896-b760-b980701054c4)

If you mean services like YouTube, it is because your server IP is marked as a server, not just residential IPs (like the IPs that you get from an ISP). 

I think this problem is related to the business model of the websites. For example, YouTube may be doing this to avoid (potentially) wasting bandwidth on scraping bots. 

Anyway i don't think this issue is related to hysteria. If you want to follow up on this problem, you should contact YouTube support or...
